base_company_reporting/models/res_company.py

```diff
@@ -7,11 +7,12 @@
 class ResCompany(models.Model):
     _inherit = "res.company"
 
-    remit_to_info = fields.Html(
+    remit_to_info = fields.Text(
         "Remit-to Information",
         help="Remit-to information that should show in the reports when set. "
         "To reset the value, please update the source and translations strings "
         "of the translation record.",
         translate=True,
-        sanitize=False,
     )
+
+    contact_team = fields.Char("contact_team")
```

Comment:
```suggestion
    contact_team = fields.Char()
```
or `contact_team = fields.Char("Department")` or something like this.

If you assign a string, it will show as the field name (so, "contact_team" is strange here).  If you do not assign a string, Odoo automatically generates the name string based on the technical name (i.e. when the technical name is "contact_team", then the name string will be "Contact Team").

Reply:
Thank you, I understand and delete it.